likeminds-chat-reactnative-integration/ChatSX/lmChatProvider/type.ts

```diff
@@ -102,9 +102,6 @@ export interface LMChatProviderProps {
   children: React.ReactNode;
   userName: string;
   userUniqueId: string;
-  cohortId: string;
-  reactionListStyles?: ReactionListStylesProps;
-  chatBubbleStyles?: ChatBubbleStylesProps;
-  inputBoxStyles?: InputBoxStyles;
-  themeStyles?: ThemeStyles;
+  profileImageUrl: string;
+  lmChatInterface: any;
```

Comment:
why is this type is any?

Reply:
because type of `lmChatInterface` is not fixed, it depends what name the client gives to the class implementing our callbacks, like here it is `CustomCallbacks` but it could be different when client implements it